include/wx/bmpcbox.h

```diff
@@ -43,7 +43,7 @@ class WXDLLIMPEXP_CORE wxBitmapComboBoxBase
     virtual ~wxBitmapComboBoxBase() { }
 
     // Sets the image for the given item.
-    virtual void SetItemBitmap(unsigned int n, const wxBitmap& bitmap) = 0;
+    virtual void SetItemBitmap(unsigned int n, const wxBitmapBundle& bitmap) = 0;
```

Comment:
I hope we can indeed make this change, but it is incompatible as it breaks any code overriding this function. I think, without being 100% sure, that this function is not actually supposed to be overridden in the application code and I can't find any occurrences of it being overridden outside of wx using GitHub code search (just calls to it, which should be fine). But even so, this change still should be mentioned in the "INCOMPATIBLE CHANGES SINCE 3.0.x:" section of `docs/changes.txt`.